NewForce-Cohort-7/tabloidfullstack-thegroundbeetles
# Cm 02

Ticket # 14: https://trello.com/c/rLHFI3qk/14-14-create-a-tag

The functionality to add new tags has been added. Given a user is logged in the app when they select the Tags link in the menu they should see a Create Tag button. Once the button is pressed the user is directed to a form where they can enter a new tag name. When they press save they will be directed back to the tag list page and should see their tag on the list. 

Type of change:
*New feature (non-breaking change which adds functionality)

Testing Instructions
Reference [How to test a teammate's PR](https://gist.github.com/jordan-castelloe/96f76d5bd4cfbd2181084af674c2a1af#when-you-need-to-test-a-teammates-pr)
git checkout grTabloid19
If you don't see the most recent version of my files, git pull origin cm-02
Add all seed data by adding a new query and copying the code from the links below
Use this to fill out tables and columns:
https://github.com/NewForce-Cohort-7/tabloidfullstack-thegroundbeetles/blob/main/SQL/01_Tabloid_Create_DB.sql
Use this to seed the database:
https://github.com/NewForce-Cohort-7/tabloidfullstack-thegroundbeetles/blob/main/SQL/02_Tabloid_Seed_Data.sql
To execute your query just press Ctrl+Shift+e inside of your query.
Run the program by clicking the Green Arrow next to TabloidMVC at the top.
In the browser, please go to Login on the nav bar
Log in with this email for both the email and password: [rsandwith0@google.com.brx](mailto:rsandwith0@google.com.brx). This should direct you back to the home page.
In the nav bar select Tags
You should be directed to a page with the list of tags
From the Tags list page press the 'Create Tag' button at the top of the page
You should be directed to a form to enter a new tag name.
Enter a tag name and press the save button
You should be directed back to the tag list page and see your tag was added to the list. 
Locate Logout on the Navbar & click this to log out.

Checklist:
[ x] - My code follows the style guidelines of this project
[x ] - I have performed a self-review of my code
[x ] - I have commented my code, particularly in hard-to-understand areas
[x ] - My changes generate no new warnings or errors
[x ] - I have added test instructions that prove that my feature works
